html-aam/index.html

```diff
@@ -9365,6 +9365,132 @@ <h4 id="att-controls">`controls`</h4>
+                    <a class="core-mapping" href="#ariaExpandedFalse">`aria-expanded=false`</a>
+                  </p>
+                  <p>
+                    If the element with the `commandfor` attribute has a `command` attribute with value of "`show-popover`", or "`toggle-popover`", and the associated element is an accessibility
+                    ancestor of the element with the `commandfor` attribute:
+                    <a class="core-mapping" href="#ariaExpandedUndefined">`aria-expanded=undefined`</a>
+                  </p>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+                <div class="general"><span class="type">Object attributes:</span> `details-roles:popover`</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+                <div class="general"><span class="type">Object attributes:</span> `details-roles:popover`</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th>Comments</th>
+              <td>
+                <p>
+                  User Agents MUST expose an <a class="core-mapping" href="#ariaDetails">`aria-details`</a> relation with the associated element <strong>except</strong> under the following conditions:
+                </p>
+                <ul>
+                  <li>The element's `command` attribute value is a value other than "`show-popover`" or "`toggle-popover`",</li>
+                  <li>The associated popover element is the next immediate accessibility sibling to the invoking element,</li>
+                  <li>The element's `command` attribute is in the "`toggle-popover`" state and the element is a descendant of the `popover` it is associated with.</li>
+                  <!-- accessibility sibling will be defined in the ARIA specification -->
+                </ul>
+                <p class="note">A button that a [=button/form owner=] without `type` in the "`button`" state cannot invoke a command.</p>
+              </td>
```

Comment:
just another instance of where indicating expectations for show-modal would be needed (or not if they had their own tables, per my prior comment)